projects/novo-elements/src/elements/tooltip/Tooltip.directive.ts

```diff
@@ -40,10 +40,13 @@ export class TooltipDirective implements OnDestroy, OnInit {
   isHTML: boolean;
   @Input('tooltipCloseOnClick')
   closeOnClick: boolean = false;
+  @Input('tooltipOnOverflow')
+  overflow: boolean = false;
```

Comment:
I think it would be easier to read if the variable was also called: `tooltipOnOverflow`, just because the word `overflow` is quite overloaded and it's harder for me to distinguish what the intent is later on in the file.